Dockerfile_browser_tools.dockerfile

```diff
@@ -9,7 +9,6 @@ RUN npx puppeteer browsers install chrome
 
 # # Tell Puppeteer to skip installing Chromium. We'll be using the installed package.
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
-ENV PUPPETEER_EXECUTABLE_PATH=/home/circleci/.cache/puppeteer/chrome/linux-121.0.6167.85/chrome-linux64/chrome
```

Comment:
I'd like us to do the test which I suggested when we were working on this together, prior to merging into `main`. Can you remember what that was?

Reply:
@exonian I think we did several variations to test if we needed the ENV's. From memory we did:

- Have the same set-up in separate PRs to see if config worked replicating the CircleCI process ([PR one](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1219),  [PR two](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1221),  [PR three](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1225))
- Remove and add in the ENV, running github workflow/action in between commits, to see if ENV's were needed ([On PR two:](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1221) [02e30b0](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1221/commits/02e30b0acfdb9c2fdddbc4136dc994ae465ac595), [7cccd9e](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1221/commits/7cccd9e42e74c4596008228b3b0beabb2e03001a), [66cfd04](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1221/commits/66cfd04f779c9a8f48a0fc8e675d6a18c6e3fdfa), [8f404b6](https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1221/commits/8f404b6221268514958b6a5721a0b82d5aab0073))

Reply:
Yeah it was the idea to create a new branch off this one, just in case there's any weirdness going on that makes this branch work forever despite us removing something important.

Reply:
tested a new branch made from this branch, successfully runs tests against the new dockerfile and also against a dockerfile built from main.

Also tested another branch against the dockerfile built from this branch and it runs successfully.